pyramid/config/__init__.py

```diff
@@ -903,6 +921,13 @@ def end(self):
         """
         return self.manager.pop()
 
+    def __enter__(self):
+        self.begin()
+        return self
+
+    def __exit__(self, exc_type, exc_value, exc_traceback):
+        self.end()
```

Comment:
Were the comments here addressed? Or did we decide not to move forward with some of the comments here?